cmd/notify-elasticsearch.go

```diff
@@ -17,14 +17,20 @@
 package cmd
 
 import (
-	"encoding/hex"
-	"errors"
+	"context"
 	"fmt"
 	"io/ioutil"
+	"time"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/minio/sha256-simd"
-	"gopkg.in/olivere/elastic.v3"
+	"gopkg.in/olivere/elastic.v5"
+)
+
+var (
+	makeESError = makeNotificationErrorMaker("Elasticsearch Notifier Error")
```

Comment:
Actually we should just keep it as "Elasticsearch:" and for everything as well.  Calling it "Elasticsearch Notifier Error" is just verbose and redundant. 

Because "Elasticsearch" already means that it is notifier and "Error:" is not needed since you are encapsulating these messages inside `fmr.Errorf()` any ways. Unless we have another set of messages indicating "Success:" we don't it.